pkg/module/go_get_fetcher.go

```diff
@@ -42,9 +39,11 @@ func NewGoGetFetcher(goBinaryName string, goProxy string, fs afero.Fs) (Fetcher,
 		return nil, errors.E(op, err)
 	}
 	return &goGetFetcher{
-		fs:           fs,
-		goBinaryName: goBinaryName,
-		goProxy:      goProxy,
+		fs: fs,
+		goTool: goTool{
```

Comment:
Nesting arguments inside a private struct feels like an overkill, the strut name is `goGetFetcher` which already implies the fact that it uses a "goTool" i don't think it needs another layer of abstraction.